nemo/nemo/backends/pytorch/actions.py

```diff
@@ -1096,14 +1097,24 @@ def __extract_dynamic_axes(port_name: str, ntype: NeuralType,
                     raise ValueError(
                         f'Example input is None, but ONNX tracing was'
                         f' attempted')
-                torch.onnx.export(module, input_example, output,
+                if output_example is None:
```

Comment:
There should probably be a `with torch.no_grad()` wrapping all this, no?

Reply:
I'll put it inside export function - the use should not do "import framework" in examples

Reply:
yes, done